validator/validations/validations_test.go

```diff
@@ -267,7 +267,14 @@ func TestValidations_RemoveFlag(t *testing.T) {
 		args args
 		want Validations
 	}{
-		// TODO: Add test cases.
+		{
+			name: "test",
+			v:    Validations(255),
```

Comment:
Here you definitely need to work with Flags instead, otherwise the test becomes fragile.

Also, keep in mind what are you testing?

Having 3 flags, removing one, I should still have the two flags, but not the one I've removed. This doesn't actually test this at all.